.github/workflows/ci.yml

```diff
@@ -30,3 +30,7 @@ jobs:
       run: npm run test
     - name: Build
       run: npm run build
+    - name: Upload Coverage
+      uses: codecov/codecov-action@v3
+      with:
+        fail_ci_if_error: true
```

Comment:
Do you think it's worth setting `fail_ci_if_error` to false so codecov doesn't block PRs unnecessarily?

Reply:
I tried that on the previous PR, but it didn't unblock the PR - not sure why.  We still have the issue on this current PR.  I think we might just need to write tests?

Reply:
or I guess, codecov is still failing on this PR, but not blocking.  I can actually merge it since you've approved.